sunpy/visualization/animator/wcs.py

```diff
@@ -299,10 +303,27 @@ def plot_start_image_2d(self, ax):
 
         return im
 
+    def get_2d_plot_clip_bounds(self):
+        """
+        Update vmin/vmax when set to 'auto'.
+        """
+        if self.imshow_kwargs.get('vmax') == 'auto':
+            clip_interval = (1, 90)*u.percent
+            clip_percentages = clip_interval.to('%')
+            clip_limits = AsymmetricPercentileInterval(*clip_percentages.value)
+            vmin, vmax = clip_limits.get_limits(self.data_transposed)
+            return vmin, vmax
```

Comment:
Can these lines not be combined into one?

Reply:
Same here :sweat_smile:

Reply:
Why is this not one line of python code?